app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/UpscanConnector.scala

```diff
@@ -279,6 +282,21 @@ class UpscanConnectorImpl @Inject() (
     implicit hc: HeaderCarrier
   ): EitherT[Future, Error, Unit] = {
 
+    implicit val actorSystem       = ActorSystem()
+    implicit val actorMaterializer = ActorMaterializer()
```

Comment:
added this locally here because injecting it into the class broke 70 test suites :man_shrugging:  - if this works then can inject and fix tests.